clj-rn.conf.example.edn

```diff
@@ -26,4 +26,10 @@
  :figwheel-options {:nrepl-port 7888
                     :nrepl-middleware ["cider.nrepl/cider-middleware"
                                        "refactor-nrepl.middleware/wrap-refactor"
-                                       "cemerick.piggieback/wrap-cljs-repl"]}}
+                                       "cemerick.piggieback/wrap-cljs-repl"]}
+ ;; those options will be passed to `react-native run-*`
+ :run-options ""
+
+ ;; platform specific config. Supported only `:run-options`
+ :platform {:android {:run-options "--appPrefix debug"}}}
```

Comment:
How about having `:run-options {:android "" :ios "" :default ""}` ?

Also do platform specifics override the default?

Reply:
reasons why I made it like this:
- It is similar to re-natal
- This PR https://github.com/status-im/clj-rn/issues/9 require it
- It is easier to specify default config. Just `:run-options "something"`
- It is easier to work with it. Just merge platform config to main config `(merge config (get-in config [:platforms platform]))`

Platform specific config overrides default.

Reply:
From a user perspective having a single option makes more sense. Would this prevent the other PR?

Reply:
It requires more work to do it like this if it should be similar for all options and I need to change pr for status-react. If only for `:runs-options` then it is easier.

Reply:
>  if it should be similar for all options

What do you mean?

Reply:
For example `js-modules` option would look like this `:js-modules {:ios [] :android [] :default []}`

Reply:
Does this apply to `js-modules` too? Or is it for potential future enhancements?

Reply:
It is for the future so all options would look the same. If it is not required now I can do it only for `:run-options`.

Reply:
ok. So I do `:run-options` like this `:run-options {:android "" :ios "" :default ""}`?

Reply:
I think that's the cleanest way to do this?

Reply:
ok, I will do it like this.